_data/versions.yml

```diff
@@ -905,6 +905,41 @@ v3.3:
       url: https://github.com/projectcalico/pod2daemon/releases/tag/v3.3.0
 
 v3.2:
+- title: v3.2.8
+  components:
+    calico/node:
+      version: v3.2.8
+      url: https://github.com/projectcalico/node/releases/tag/v3.2.8
+    calicoctl:
+      version: v3.2.8
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v3.2.8
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v3.2.8/calicoctl
+    calico/cni:
+      version: v3.2.8
+      url: https://github.com/projectcalico/cni-plugin/releases/tag/v3.2.8
+      download_calico_url: https://github.com/projectcalico/cni-plugin/releases/download/v3.2.8/calico
+      download_calico_ipam_url: https://github.com/projectcalico/cni-plugin/releases/download/v3.2.8/calico-ipam
+    calico/kube-controllers:
+      version: v3.2.8
+      url: https://github.com/projectcalico/kube-controllers/releases/tag/v3.2.8
+    networking-calico:
+      version: 3.2.0
+      url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=3.2.0
```

Comment:
Same as in other PR, this is not a browseable URL.

Reply:
I'll fix this as per https://github.com/projectcalico/calico/pull/2802/files#r316377866

Reply:
https://github.com/projectcalico/calico/pull/2802/commits/2bd86f89dc074781cfbfba3d50c3230544b50eba

Reply:
I think this still needs to be updated here. Right?